CMakeLists.txt

```diff
@@ -47,6 +47,8 @@ project(${PROJECT_NAME} ${PROJECT_ARGS})
 check_minimal_cxx_standard(14 ENFORCE)
 
 # Project dependencies
+set(CMAKE_MODULE_PATH "${CMAKE_SOURCE_DIR}/cmake/find-external/CppAD/"
+                      ${CMAKE_MODULE_PATH})
```

Comment:
Unfortunately, it needs this when we install Pinocchio with CppAD (at least with my setup, see below). This is a workaround, but I agree with you. It would be great if we don't have to do this

My setup is:
  - Pinocchio installed from source.
  - CppAD(CodeGen) installed from Conda.
  
@jcarpent -- I apologise if I tag you in this thread. I think you might help us clarify this point and look for a way forward.

Reply:
I'll try to reproduce the issue, and open the PR in the correct place for this